cmd/kubeadm/app/util/staticpod/utils_test.go

```diff
@@ -131,6 +150,7 @@ func TestGetEtcdProbeEndpoint(t *testing.T) {
 						"listen-metrics-urls": "https://1.2.3.4:1234,https://4.3.2.1:2381"},
 				},
 			},
+			isIPv6:           false,
```

Comment:
nit: `false` is the default value for a boolean struct member.
but this is fine like that too.